muselog/datadog.py

```diff
@@ -72,8 +72,15 @@ def default(self, obj):
             return "traceback"
         elif isinstance(obj, timedelta):
             return obj.__str__()
-
-        return super().default(obj)
+        else:
```

Comment:
Is there a library that provides a very robust JSONEncoder? I know this is an issue that plagues many python devs, so I suspect someone has created a library that can encode /most/ stdlib objects. This wouldn't eliminate the need for a fallback like you have, but it should be a marked improvement.

Reply:
I will check further.

Reply:
I couldn't find any custom library for python JSON encoder, as most scripts just try to take care of specific types to encode, similar to what we do too.

Reply:
Okay.

Reply:
I was searching earlier and couldn't find much either, it's lame